packages/dataviews/src/components/dataviews-bulk-actions/index.tsx

```diff
@@ -155,27 +206,33 @@ interface ToolbarContentProps< Item > {
 	data: Item[];
 	actions: Action< Item >[];
 	getItemId: ( item: Item ) => string;
+	picking?: boolean;
 }
 
 function ActionTrigger< Item >( {
 	action,
 	onClick,
 	isBusy,
 	items,
+	isTextButton,
```

Comment:
I think we might have already discussed this, but will we ever want to support other variants? I.e. would it be worth having a `buttonType` or `buttonVariant` prop that uses a string, to support different kinds of buttons? Or do we think a simple boolean is good enough for now / most cases?

Reply:
Yeah, it's a tricky one. The underlying `Button` already has the concept of variants (primary / secondary / tertiary), so I'm not sure if it would be confusing for actions to have a similar prop but that does something different.

Compared to the previous time I tried this, this particular change is internal only, so I think `isTextButton` is ok for now and can be revised without needing a major release.

Reply:
> Compared to the previous time I tried this, this particular change is internal only, so I think isTextButton is ok for now and can be revised without needing a major release.

Agreed. Thanks for the discussion here! I'll resolve this thread.